### ISSUE TITLE
Apkmonk support - apkmonk.com

### ISSUE THREAD

**Issue by luxzg** [hemker/apkgrabber#74]
Hi,

I stumbled on your app while looking for fixes to Huawei/Google situation, as someone linked you in APKUpdater GitHub. I posted this in [upstream issue #308](https://github.com/rumboalla/apkupdater/issues/308#issue-591804090), but I'm adding info here for visibility as well.

Explanation:
- Huawei has new app "AppSearch" (AppSuche) that directs users without Google Services to alternative download sites
- list of download pages include APKPure, APKmirror and apkmonk
- APKUpdater & APKGrabber apps nicely provide update functionality for the first two
- by adding apkmonk.com it would cover all sources that AppSearch currently uses

In short - please add https://www.apkmonk.com/ to supported sites in your app, that way future millions of Huawei users will be able to replace Google Play just by using 2 apps : AppSearch to find and install APKs, and APKUpdater / APKGrabber to keep them updated.

Thanks for your time! :+1:

**Comment by hemker** [comment]
@luxzg will take a look at it for the next feature update. Having a first look on apkmonk, the crawling and page processing shouldn't be that complicated :+1:

**Comment by Bluscream** [comment]
Any news for this?
